backend/chainlit/session.py

```diff
@@ -3,25 +3,13 @@
 import mimetypes
 import shutil
 import uuid
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Callable,
-    Deque,
-    Dict,
-    List,
-    Literal,
-    Optional,
-    Union,
-)
+from typing import TYPE_CHECKING, Any, Callable, Deque, Dict, Literal, Optional, Union
 
 import aiofiles
 from chainlit.logger import logger
 
 if TYPE_CHECKING:
-    from chainlit.message import Message
-    from chainlit.step import Step
-    from chainlit.types import FileDict, FileReference
+    from chainlit.types import FileDict, FileReference  # noqa: F401
```

Comment:
Referenced in the comment on line 77.

Reply:
Again; I really want to move everything to `: <type>` modern Python types.

Reply:
Done ✅